spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -33,13 +33,16 @@ def get_parser():
 
               - volume `[mm^3]`: volume of the lesion
               - length `[mm]`: maximal length along the Superior-Inferior (SI) axis across all slices
+              - width `[mm]`: maximal width along the Anterior-Posterior (AP) axis across all slices
               - max_equivalent_diameter `[mm]`: maximum diameter of the lesion, when approximating the lesion as a circle in the axial plane
               - max_axial_damage_ratio `[]`: maximum ratio of the lesion area divided by the spinal cord area
-              - midsagittal_spinal_cord_slice: midsagittal slice number of the spinal cord defined based on the spinal cord segmentation
```

Comment:
It looks like [some of the tests](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/11939714655/job/33280820096?pr=4702#step:5:1941) are failing because we got rid of this key:

```
 KeyError: 'midsagittal_spinal_cord_slice'
```

If I Ctrl+F the project for this key, I find a number of instances that will need to be updated:

![image](https://github.com/user-attachments/assets/ceb54359-ae67-4bb0-b5cb-ccf0f341cfea)

Reply:
good catch. fixed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4702/commits/46fa51a4015bd019cfd8a21a19b44e6db778ccaa